### PR TITLE
update status bar on window focus

### PR DESCRIPTION
**Pull request by xavluiz**
update the status bar when one of the vscode window focuses to help keep the windows status metrics in sync

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -15,7 +15,6 @@ import { softwarePost } from "./http/HttpClient";
 import { configureSettings, showingConfigureSettingsPanel } from "./managers/ConfigManager";
 import { initializeStatusBar } from "./managers/StatusBarManager";
 import { SummaryManager } from "./managers/SummaryManager";
-import { SyncManager } from "./managers/SyncManger";
```
Comment (mmclead) [inline src/extension.ts:18]: if we aren't going to use the SyncManager, you'll want to remove the source too.

File: src/managers/KpmManager.ts [hunk 2]
```diff
@@ -95,6 +96,9 @@ export class KpmManager {
     if (event.focused) {
       PluginDataManager.getInstance().editorFocusHandler();
       this.tracker.trackEditorAction("editor", "focus");
+      // update the status bar only as this is now the primary window and the
+      // previous primary window may have updated the session summary data.
+      updateStatusBarWithSummaryData();
```
Comment (mmclead) [inline src/managers/KpmManager.ts:101]: why did the syncManager not work?    I get this approach, but its not as ideal as the other

File: src/extension.ts (hunk 1 above)
Comment (xavluiz) [reply src/extension.ts:18]: yeah, i forgot to check if it was used anywhere else. removing it now

File: src/managers/KpmManager.ts (hunk 2 above)
Comment (xavluiz) [reply src/managers/KpmManager.ts:101]: oh this should. I forgot we had this. I tested and it didn't register the change, but I'll get this back in and test one more time

**Review by mmclead — APPROVED** [review]

**Pull request merged**